### PR TITLE
Adds husky to project

### PR DESCRIPTION
**Pull request by GoldenCherro**

### CODE REVIEW

File: cherro-front/package.json [hunk 1]
```diff
@@ -38,6 +38,14 @@
   },
   "jest": {
     "clearMocks": true
+  },
+  "devDependencies": {
```
Comment (mlvieras) [inline cherro-front/package.json:42]: 👏

File: cherro-front/package.json [hunk 2]
```diff
@@ -38,6 +38,14 @@
...
+    "husky": "^4.3.5"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm test",
```
Comment (mlvieras) [inline cherro-front/package.json]: We should avoid this as a pre-commit hook. It will complicate development.

File: cherro-front/package.json [hunk 3]
```diff
@@ -38,6 +38,14 @@
...
+      "pre-push": "npm test"
```
Comment (mlvieras) [inline cherro-front/package.json]: Please also add the linter here

File: cherro-front/package.json (hunk 2 above)
Comment (GoldenCherro) [reply cherro-front/package.json]: pre-commit deleted

File: cherro-front/package.json [hunk 4]
```diff
@@ -5,20 +5,20 @@
   "dependencies": {
     "axios": "^0.19.2",
     "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.5",
+    "enzyme-adapter-react-16": "^1.15.2",
```
Comment (mlvieras) [inline cherro-front/package.json]: Why was the adapter downgraded?

File: cherro-front/package.json [hunk 5]
```diff
@@ -5,20 +5,20 @@
...
     "eslint-config-airbnb": "^18.2.1",
     "history": "^4.10.1",
     "node-sass": "^4.14.1",
     "prop-types": "^15.7.2",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-router-dom": "^5.2.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-router-dom": "^5.1.2",
```
Comment (mlvieras) [inline cherro-front/package.json]: I don't think we need to downgrade this

File: cherro-front/package.json [hunk 6]
```diff
@@ -37,7 +37,12 @@
     ]
   },
   "jest": {
-    "clearMocks": true
+    "clearMocks": true,
+    "coverageThreshold": {
```
Comment (mlvieras) [inline cherro-front/package.json]: I think this makes sense, but why did you add it?

Comment (GoldenCherro) [reply cherro-front/package.json]: it was one of the possible solutions i read to the test problem

Comment (mlvieras) [reply cherro-front/package.json]: Oh, then let's remove it 😄 Unless you anticipate any issues

**Review by mlvieras — APPROVED** [review]
Nice work 😄
